### ISSUE TITLE
Enabling both stream_buffer and stream_buffer_remote

### ISSUE THREAD

**Issue by Massimo-B** [digint/btrbk#312]
Beside any recommendation for or against mbuffer (you said it's another point of failure), is it recommended to have both **stream_buffer** and **stream_buffer_remote** when using **volume ssh://...**?

On --progress -v I see
```
in @ 8183 kiB/s, out @ 8183 kiB/s, 33.2 GiB total, buffer 100% fulll
```
which I guess is the local mbuffer output from option **stream_buffer**. Is it possible to also show the remote buffer status?

When comparing all the stream_buffer* and **stream_compress**, how do you compare the throughput? As the mbuffer rate is very jumpy. The only real comparison I guess would be the total time for a complete transfer.

**Comment by digint** [comment]
> Beside any recommendation for or against mbuffer (you said it's another point of failure), is it recommended to have both stream_buffer and stream_buffer_remote when using volume ssh://...?

Added some text in: d874a7789ac7137109688c60fb79cad12bbf4849 documentation: btrbk.conf.5: explain stream_buffer_remote

tl;dr it depends on many factors

> Is it possible to also show the remote buffer status?

Not directly. I think its possible to define a logfile in `~/.mbuffer.rc`, see mbuffer(1).

> When comparing all the stream_buffer* and stream_compress, how do you compare the throughput? As the mbuffer rate is very jumpy. The only real comparison I guess would be the total time for a complete transfer.

Yes, sadly it is very jumpy some times... note that mbuffer also prints some useful stats after a transfer (with --progress)
